Pipfile

```diff
@@ -0,0 +1,21 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+
+[dev-packages]
+
+"e1839a8" = {path = ".", editable = true}
```

Comment:
What is `"e1839a8"` here? It looks suspiciously like a git hash and also very much like the line from the docs about using `-e .`. Surely it's not a magic constant baked into pipenv?